src/sources/metadatasourcetaglib.cpp

```diff
@@ -30,6 +34,14 @@ const QString kSafelyWritableTempFileSuffix = QStringLiteral("_temp");
 // potential failures caused by exceeded path length.
 const QString kSafelyWritableOrigFileSuffix = QStringLiteral("_orig");
 
+#ifdef __WINDOWS__
+// After Mixxx has closed the track file, the indexer or virus scanner
```

Comment:
This comment belongs to the code where these constants are actually used, but not to these constants. Mentioning why ReplaceFileW() is needed here but not in the context were it is actually invoked is confusing.